aiqleads/scripts/run.py

```
@@ -71,5 +75,12 @@ def test_project_tracking():
         return False
 
 if __name__ == "__main__":
-    success = test_project_tracking()
-    sys.exit(0 if success else 1)
+    try:
+        success = test_project_tracking()
+        sys.exit(0 if success else 1)
+    except KeyboardInterrupt:
+        logger.info("Testing interrupted by user")
+        sys.exit(1)
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)